.github/actions/cdn_code_deploy/action.yaml

```diff
@@ -0,0 +1,107 @@
+name: CDN Code Deploy 
+description: Action that Deploy an artifact to the CDN
+
+# This action allows you to deploy the passed artifact to the CDN.
+# The artifact have to be a directory that contains all the files you want to sync.
+
+inputs:
+  inputs:
+    resource_group_name:
+      description: CDN resource group name.
+      type: string
+      required: true
+    storage_account_name:
+      description: Storage account name.
+      type: string
+      required: true
+    profile_cdn_name:
+      description: CDN Profile name.
+      type: string
+      required: true
+    endpoint_name:
+      description: CDN endpoint name.
+      type: string
+      required: true
+    blob_container_name:
+      description: Blob container name.
+      type: string
+      required: true
+    content_paths:
+      description: If you want to include only specific files set 'true' else 'false'
```

Comment:
what do you mean here with "specific files"?

Reply:
You can decide to purge the entire folder or only the files that have been added and/or modified